pdf_on_submit/attach_pdf.py

```diff
@@ -25,6 +25,8 @@ def attach_pdf(doc, event=None):
 
 
 def process_enabled_doctype(doc, settings, in_background):
+	DEFAULT_TIMEOUT = 30 #Fallback if no timeout is set
```

Comment:
```suggestion
	DEFAULT_TIMEOUT = 30
```

No need to comment the obvious, this is already implied in the name of the constant.